hearpreprocess/pipeline.py

```diff
@@ -444,6 +485,17 @@ def split_train_test_val(self, metadata: pd.DataFrame):
             train_percentage + valid_percentage + test_percentage == 100
         ), f"{train_percentage + valid_percentage + test_percentage} != 100"
 
+        diagnostics.info(
+            f"{self.longname} Split percentage for the splitting: "
```

Comment:
"Split percentage for the splitting" is pretty redundant

"Split percentage"

Reply:
"Split percentage for splitting the train set to generate other sets". I this fine?